src/backend/routers/paras.test.ts

```diff
@@ -2,19 +2,119 @@ import test from "ava";
 import { getTestServer } from "backend/tests";
 
 test("getParaById", async (t) => {
-  const { trpc, db } = await getTestServer(t, { authenticateAs: "para" });
-
-  const { user_id } = await db
-    .insertInto("user")
-    .values({
-      first_name: "John",
-      last_name: "Doe",
-      email: "john.doe@example.com",
-      role: "para",
-    })
-    .returningAll()
-    .executeTakeFirstOrThrow();
-
-  const para = await trpc.getParaById.query({ user_id });
-  t.is(para.user_id, user_id);
+  const { trpc, seed } = await getTestServer(t, { authenticateAs: "admin" });
+
+  const para = await trpc.getParaById.query({ user_id: seed.para.user_id });
+  t.is(para.user_id, seed.para.user_id);
+});
+
+test("getMyParas", async (t) => {
+  const { trpc, db, seed } = await getTestServer(t, {
+    authenticateAs: "admin",
```

Comment:
This doesn't really need 'admin' privileges - consider changing this to 'para' (to be changed to 'staff' soon)

Reply:
Creating another entry in the db, either in this test (say using createPara) or in the seed db sounds good. In general, entities should have the lowest privilege needed for fulfilling their duties.